Master-Game/World/TileFactory.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MasterGame.World
+{
+    public class TileFactory
+    {
+        public static BaseTile CreateTile(TileType tileType)
+        {
+            switch (tileType)
+            {
+                case TileType.Grass:
+                    return new GrassTile(1, 1);
```

Comment:
once you sync with master you wont need the (1, 1) since @wilsont3 remove the length, width requirement

Reply:
How is stuff like this normally handled? I assume that once we decided to merge the commits from this branch into the master, GitHub will notify us that there are conflicts. 

Is that handled mostly through GitHub? Or is that something that we would handle by pulling down down the merge, commit the changes, and then push those changes back to the repo? Or is it something different?

Reply:
you can handle it locally. Let's do it next session so we get it right.

Reply:
Sounds good.